jasmine/spec/string-extension-tests.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+describe('String Extension', function() {
+
+  describe('Has Vowels', function() {
+    it('should return true if the string contains vowels', function() {
+      expect('hi'.hasVowels()).toBeTruthy();
+      expect('HI'.hasVowels()).toBeTruthy();
+    });
+  });
+
+  describe('To Upper', function() {
+    it('should return the string passed as all upper cases characters.', function() {
+      expect('hi'.toUpper()).toEqual('HI');
+      expect('HI'.toUpper()).toEqual('HI');
+    });
+  });
+
+  describe('To Lower', function() {
+    it('should return the string passed as all lower cases characters.', function() {
+      expect('HI'.toLower()).toEqual('hi');
+      expect('hi'.toLower()).toEqual('hi');
+    });
+
+  });
+
+  describe('Upper Case First Letter', function() {
+    it('should return the string passed with the First Character in Upper case.', function() {
+      expect('hi'.ucFirst()).toEqual('Hi');
+      expect('Hi'.ucFirst()).toEqual('Hi');
+      expect('HI'.ucFirst()).toEqual('Hi');
+    });
+  });
+
+  describe('is Question', function() {
+    it('should return true if the string is a question (ending with a question mark).', function() {
+      expect('hi?'.isQuestion()).toBeTruthy();
+      expect('hi? am good'.isQuestion()).toBeFalsy();
+    });
+  });
```

Comment:
Does '?'.isQuestion() return true? If it does, then the test is not conclusive. It should return false.